src/app/views/query-response/QueryResponse.tsx

```diff
@@ -64,6 +67,22 @@ class QueryResponse extends Component<IQueryResponseProps, {}> {
     ];
 
     if (mode === Mode.Complete) {
+      pivotItems.push(
+        <PivotItem
+          key = 'adaptive-cards'
```

Comment:
Do we only want to show adaptive cards for the complete graph explorer experience?

Reply:
For now I think we can treat this just as the snippets tab and make any changes based on any new feedback.